### PR TITLE
Add verify-deps job to the rest of master team operators

### PR DESCRIPTION
**Pull request by tnozicka**
/cc @deads2k @sttts

### CODE REVIEW

**Comment by tnozicka** [comment]
/hold
(for https://github.com/openshift/release/pull/4136)

**Comment by tnozicka** [comment]
`verify-deps` is now optional to avoid cases where the remote repository is down and the inability for approvers to force `/override`

File: ci-operator/config/openshift/cluster-kube-apiserver-operator/openshift-cluster-kube-apiserver-operator-master.yaml [hunk 1]
```diff
@@ -32,15 +32,19 @@ test_binary_build_commands: make build GOFLAGS='-race'
 tests:
 - as: verify
   commands: |
-    go get -u golang.org/x/tools/imports
     make verify -k
   container:
     from: bin
+- as: verify-deps
+  commands: |
+    HOME=$( mktemp -d )
```
Comment (sttts) [inline ci-operator/config/openshift/cluster-kube-apiserver-operator/openshift-cluster-kube-apiserver-operator-master.yaml]: why?

Comment (tnozicka) [reply ci-operator/config/openshift/cluster-kube-apiserver-operator/openshift-cluster-kube-apiserver-operator-master.yaml]: something needed it set (I think it was glide, for the cache) but the base images don't set it

**Comment by tnozicka** [comment]
/hold cancel
(https://github.com/openshift/release/pull/4136 merged)

**Comment by tnozicka** [comment]
@ravig fyi, we have talked about it before on slack and as the check is optional and scheduler now owned by our team, I am gonna approve it for scheduler as well

File: ci-operator/config/openshift/cluster-kube-apiserver-operator/openshift-cluster-kube-apiserver-operator-master.yaml (hunk 1 above)
Comment (ravisantoshgudimetla) [reply ci-operator/config/openshift/cluster-kube-apiserver-operator/openshift-cluster-kube-apiserver-operator-master.yaml]: Ya, to be clear, while verifying deps, we will pull the vendored packages the to `$HOME` and do a comparison. So, we need to set it.

**Pull request merged**